tests/tasks/addons/fluentbit.yaml

```diff
@@ -48,4 +48,5 @@ spec:
       EOL
       kubectl apply -f ./fluentbit-configmap.yaml
       kubectl apply -f https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/latest/k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluent-bit/fluent-bit.yaml
-      kubectl rollout status daemonset fluent-bit -n amazon-cloudwatch --timeout 5m
+      kubectl patch daemonset fluent-bit -n amazon-cloudwatch -p='{"spec":{"template": {"spec":  {"tolerations": [{"key": "monitoring", "operator": "Equal", "value": "true", "effect": "NoSchedule"}]}}}}'
+      kubectl rollout status daemonset fluent-bit -n amazon-cloudwatch --timeout 60m
```

Comment:
60mins is too long of a time. Why did you decide on `60` ?

Also, we should actually install fluent-bit before `mng` creation [here](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/pipelines/eks/awscli-cl2-load-with-addons.yaml#L119-L120), that way it doesn't have to wait for 5k/10k/15k DS pods to come up depending on our scale test size so deployment status is much more predictable that way.